methods/transient/Transient_base.ipynb

```diff
@@ -46,19 +46,25 @@
     "NISAR must be able to constrain displacements in two look directions (left-looking on both ascending and descending tracks) for the active area of 70% of target sites with an accuracy that scales with baseline distance L between any two locations within a scene. This pertains to all directly measured or inferred 12-day interferograms over the duration of the mission. Here, accuracy is calculated using L in kilometers but with the units removed, at or better than 100 m resolution. The 12-day time scale corresponds to a repeat NISAR pass on each of the ascending and descending satellite tracks, from which interferograms can be made and displacements estimated. The NISAR mission has compiled a list of 2000 global targets covering areas of known or potential transient deformation related to the processes specified in the requirement (NISAR Handbook Appendix H, 2018). These targets include all Earth's active volcanoes above sea-level, areas of rapid glacial mass changes, selected deforming reservoirs of water, oil, gas, CO2 and steam and landslide-prone areas near major population centers, as well as sites where selected disaster-related events have occurred. \n",
```

Comment:
<p>For the "Transient_approach2.1/2.ipynb" and all the other similar cases, I would recommend adding a link to the file, in mark down syntax as below. Then one could click it to jump to the notebook directly.</p><p> </p><pre class="ql-syntax" spellcheck="false">[Transient_approach2.1.ipynb](./Transient_approach2.1.ipynb)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
I have changed it.